internal/local/app.go

```diff
@@ -208,6 +214,20 @@ func (a App) WriteConfig() error {
 	return WriteFile(filepath.Join(a.RootDir, a.Config.String()), 0666, bytes.NewReader(data))
 }
 
+// WriteCliConfig writes the .mdb/app_meta.json config to the specified path given a remote app and config version
+func (a App) WriteCliConfig() error {
+	data, err := MarshalJSON(a.AppMeta)
```

Comment:
`AppMeta` is a struct so it should never be nil. I could add a check to make sure that `config_version` is not empty but as we discussed it may not matter too much and/or be the responsibility of the caller.